capstone/src/main/java/com/google/sps/servlets/LessonHandlerServlet.java

```diff
@@ -71,7 +54,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
             }
         }
         catch (EntityNotFoundException e) {
-            System.out.println("Lesson entities don't exist.");
+            System.err.println("Lesson entities don't exist.");
```

Comment:
Printing additional context information helps when you are trying to debug a problem by looking at logs. So you could also print out the room ID which was expected to exist and maybe the room name or something